com.unity.render-pipelines.universal/Editor/ScriptableRendererDataEditor.cs

```diff
@@ -1,14 +1,142 @@
 using System;
 using System.Collections.Generic;
+using System.Collections.Specialized;
+using System.Linq;
 using System.Reflection;
 using System.Text.RegularExpressions;
 using UnityEngine;
+using UnityEngine.Rendering;
 using UnityEngine.Rendering.Universal;
 using UnityEngine.Scripting.APIUpdating;
 using Object = UnityEngine.Object;
 
 namespace UnityEditor.Rendering.Universal
 {
+    static class RemoveRendererDataDialogText
+    {
+        public static readonly string title = "Remove Renderer Data from URP Asset";
+        public static readonly string message = "This change might change the renderer data list entries";
+        public static readonly string proceed = "Proceed";
+        public static readonly string ok = "Ok";
+        public static readonly string cancel = "Cancel";
+    }
+
+    public class AssignToRendererDataWindow : EditorWindow
```

Comment:
- I think it would be better if this class is moved to it's own file, also I do not see dependencies on Universal, if we use the base class `RenderPipelineAsset`, so we can move it to core directly.

- I opened the window, and it has a lot of height, that it wasn't usefull. What about using a [PopupWindow](https://docs.unity3d.com/ScriptReference/PopupWindow.html)? Where you can calculate all the Toggle + the Buttons and return the proper height?

- Missing documentation for public class